tools/scylla-sstable.cc

```diff
@@ -996,7 +996,7 @@ void scrub_operation(schema_ptr schema, reader_permit permit, const std::vector<
             throw std::invalid_argument("missing mandatory command-line argument --scrub-mode");
         }
         const auto mode_name = vm["scrub-mode"].as<std::string>();
-        auto mode_it = boost::find_if(scrub_modes, [&mode_name] (const std::pair<std::string, compaction_type_options::scrub::mode>& v) {
+        auto mode_it = std::ranges::find_if(scrub_modes, [&mode_name] (const std::pair<std::string, compaction_type_options::scrub::mode>& v) {
```

Comment:
Nitpick:
Here and in a few other places you replaced `boost::find_if` with `std::ranges::find_if`.
You could replace instead with `std::find_if` (no ranges involved) - but I can see the benefit of using one algorithm in all places (especially when it consumes the other).

Reply:
i don't see why it's preferred to use `std::find_if` here.